draft-ietf-rats-architecture.md

```diff
@@ -243,7 +243,7 @@ Typically, a solution starts with a specific component (sometimes referred to as
 provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.
 Such components perform operations that help determine the trustworthiness of yet other components,
 by collecting, protecting or signing measurements.
-Measurements that have been signed by such components comprise Evidence that when evaluated either supports or refutes a claim of trustworthiness.
+Measurements that have been signed by such components comprise of Evidence that when evaluated either supports or refutes a claim of trustworthiness.
```

Comment:
My reading of https://www.merriam-webster.com/dictionary/comprise is that this change is not grammatically correct.  It was ok as it was, but more readable would be
```suggestion
Measurements that have been signed by such components are comprised of Evidence that when evaluated either supports or refutes a claim of trustworthiness.
```
That is, "of" is used with "are comprised" but not "comprise".